### ISSUE TITLE
Charting guide for Grafana

### ISSUE THREAD

**Issue by jessover9000** [grafana/grafana#20321]
Create charting guide for Grafana which is similar to the one from Google/material design (https://material.io/design/communication/data-visualization.html).

**Outcomes**
- [ ] docs page and/or blog post
- [ ] visuals like screenshots and illustrations to explain constraints and recommendations

**Comment by oddlittlebird** [comment]
@jessover9000, what is the status of this?

**Comment by jessover9000** [comment]
> @jessover9000, what is the status of this?

Unfortunately this was never picked up. :( I still think it would be great to do so, especially now that the frontend-platform team is planning to tweak and improve some of the visualizations. Also @hemdrup already started working on some general visualization improvements like the legend, so there is potential. Nico, could we make this part of the design system work?

**Comment by hemdrup** [comment]
@jessover9000 Yes. There will be new graph colors for G8, and then properly a deeper dive into charts after G8

**Comment by oddlittlebird** [comment]
In that case, I recommend unassigning it from you, @jessover9000 . That way someone else might pick it up.
